tests/foreman/conftest.py

```diff
@@ -51,6 +53,13 @@ def pytest_namespace():
     }
 
 
+def _extract_setup_class_ids(item):
+    setup_class_method = getattr(item.parent.obj, 'setUpClass')
+    if setup_class_method:
+        return getattr(setup_class_method, 'bugzilla_ids', [])
```

Comment:
You don't need the if block even when the setup_class_method is None:

```python
>>> setup_class_method = None
>>> getattr(setup_class_method, 'bugzilla_ids', [])
[]
```

This will make this function a two liner